master/buildbot/db/migrate_utils.py

```diff
@@ -15,8 +15,8 @@
 
 
 import sqlalchemy as sa
-
 from future.utils import text_type
```

Comment:
I'd put `future.utils` *before* `sqlalchemy` and definitely would not put them in the same group.